terraform/cp4ba/main.tf

```diff
@@ -67,11 +74,54 @@ module "install_db2" {
   ENTITLEMENT_REGISTRY_USER_EMAIL = var.entitled_registry_user
 }
 
-  # ------ D
+resource "null_resource" "create_DB_Schema" {
+
+  depends_on = [
+    module.install_db2
+  ]
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createAPPDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createBASDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createBAWDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createDBSchema.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createGCDDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createICNDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createOSDB.sh"
+  }
+
+  provisioner "local-exec" {
+    command = "${path.module}/db2_schema/createUMSDB.sh"
+  }
+}
+
+  # ------ DB2 -------
 module "install_cp4ba"{
-  source = "git::https://github.com/jgod1360/terraform-ibm-cloud-pak/tree/cp4ba/modules/cp4ba"
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/joel_cloud_pak_terraform_issue220/modules/cp4ba"
+//  source = "../../../terraform-ibm-cloud-pak/modules/cp4ba"
+    depends_on = [
+    null_resource.create_DB_Schema
+  ]
 
-  CLUSTER_NAME_OR_ID      = var.cluster_name_or_id
+  CLUSTER_NAME_OR_ID      = var.cluster_id
   cluster_config_path     = data.ibm_container_cluster_config.cluster_config.config_file_path
 
   # ---- Platform ----
```

Comment:
In the 
```
data "external" "get_endpoints" {
  count = var.enable ? 1 : 0
```
`var.enable` is referring to the db2 module in the variables.tf file.  Need to fix this.  We probably need to know if they want the db2 module to provision so there should be an `enable_db2` variable as well as an `enable_cluster`.  We don't need an `enable_cp4ba` anymore as this script as it is assumed they want to install cp4ba or they wouldn't be calling this script.